contrib/mpi-proxy-split/two-phase-algo.cpp

```diff
@@ -130,7 +130,8 @@ TwoPhaseAlgo::commit_begin(MPI_Comm comm)
   // Set state again incase we returned from beforeTrivialBarrier
   MPI_Request request;
   MPI_Comm realComm = VIRTUAL_TO_REAL_COMM(comm);
-  int flag = 0;
+  // Consider removing the #if 0 segment below? 
```

Comment:
I don't know what's the purpose of the `#if 0` bloc below, may keep it for debugging purposes.

But in C++, we usually define the variable right before we use it, so you can move this line to 145 if we don't want to remove that `#if 0` block.